conda-store-server/hatch_build.py

```diff
@@ -61,10 +61,12 @@ def initialize(self, version: str, build_data: Dict[str, Any]) -> None:
             # main.js to enable easy configuration see
             # conda_store_server/server/templates/conda-store-ui.html
             # for global variable set
-            with (source_directory / "main.js").open("r") as source_f:
+            with (source_directory / "main.js").open("r", encoding="utf-8") as source_f:
```

Comment:
Why was the encoding change necessary?

Reply:
It was using Windows cp1252 encoding for me, which was causing errors because these Javascript files contain emojis.